common/static/common/templates/components/paging-header.underscore

```diff
@@ -4,8 +4,8 @@
     </div>
     <nav class="pagination pagination-compact top">
```

Comment:
All landmark elements, like ```<nav>``` should have an aria-label attribute so they can be distinguishable by screen readers when presented out of context, in a list of landmarks.  A good one here would be ```aria-label="Compact Pagination"```